tests/prediction_pipeline/test_postprocessing.py

```diff
@@ -13,8 +13,6 @@ def test_binarize_postprocessing():
     threshold = 0.5
     exp = xr.DataArray(np_data > threshold, dims=axes)
 
-    for dtype in ("float32", "float64", "uint8", "uint16"):
```

Comment:
Don't remove the test for different dtypes here. We still need to ensure that postprocessing yields the correct dtype that is specified in the output tensor spec.

Reply:
the helpers `make_pre/postprocessing` do not well into the new code, thus `CombinedProcessing` now appends an `EnsureDtype` processing at the end. (Maybe we want to add that one to the spec as well so it can be used between processing steps as well?
I'll add a test for the output dtype using `CombinedPreprocessing`

Reply:
> Maybe we want to add that one to the spec as well so it can be used between processing steps as well?

I don't know what you mean by this, but I would propose to do spec changes later and focus on implementing the current spec in this PR ;).

Reply:
yes of course. I mean that we might want to think about adding another processing to the documented bioimageio.spec: `EnsureDtype` or better `CastDtype`. Here we use the (secret) `EnsureDtype` Processing to ensure that at the end of postprocessing the tensor is cast to the specified dtype. 
What is new is that we do not cast in-between postprocessing steps anymore (and I don't think that is required).

Reply:
> `EnsureDtype` or better `CastDtype`. Here we use the (secret) `EnsureDtype` Processing to ensure that at the end of postprocessing the tensor is cast to the specified dtype.

Yes, we can think about adding this to the spec at some point.

> What is new is that we do not cast in-between postprocessing steps anymore (and I don't think that is required).

As far as I can recall, we didn't cast in between in the previous implementation either and would just cast at the end.